### DISCUSSION TITLE
UGS Not opening correctly?

### DISCUSSION THREAD

**Discussion by Fireball1962** [winder/Universal-G-Code-Sender#2367] [Help]
I recently downloaded the new version of UGS and when I opened it, it would get to the opening screen with the progress bar about 2/3rds done, and then quit. No errors or other indication. I searched around and found some people having the same issue and the recommended fix was to delete the cache and any other UGS related files. I did this, now when I open UGS, it come to a screen that looks nothing like UGS. I am using windows10. Here's a picture:
![UGS](https://github.com/winder/Universal-G-Code-Sender/assets/151225541/6ce42c4a-5be5-4550-976c-e0aa364a3530)

**Comment by breiler** [comment]
This is probably caused by a coruppted cache, please try the latest version here:
https://github.com/winder/Universal-G-Code-Sender/releases/tag/v2.1.2

It should automatically clear the cache on startup and will hopefully resolve the issue.

**Reply by Fireball1962** [reply]
Thanks for the reply! V2.1.2 does not load. It gets to about the 2/3's mark on the red bar on the opening screen and then just quits. The picture in the original post is from v 2.0.21.

**Reply by breiler** [reply]
Ok, then this was not the cache problem, bummer. Without your log file I'd have to guess that your settings have been corrupted.

Delete the file:
C:\Users\<username>\.ugs\UniversalGcodeSender.json

Restart and see if that solves the problem.

If that doesn't help delete the directory:
C:\Users\<username>\AppData\Roaming\ugsplatform

**Reply by Fireball1962** [reply]
I've deleted the C:\Users<username>.ugs\UniversalGcodeSender.json file and restarted. Still the same problem. 

I didn't have C:\Users<username>\AppData\Roaming\ugsplatform in <Users> (There wasn't a <AppData> directory). I did find a directory C:\Users<username>\.ugsplatform. I deleted that, restarted, and retried with no luck. 

How can I get a log file? 

Thanks for all your help. It's much appreciated.

**Reply by breiler** [reply]
C:/Users/[your username>]/AppData/Roaming/ugsplatform/var/log/messages.log

**Reply by Fireball1962** [reply]
I do not have a C:/Users/[your username>]/AppData/. There is nothing with that directory name in <users>.

**Reply by breiler** [reply]
The directory might be hidden:
https://support.microsoft.com/en-us/windows/view-hidden-files-and-folders-in-windows-97fbc472-c603-9d90-91d0-1166d1d9f4b5#WindowsVersion=Windows_10

**Reply by Fireball1962** [reply]
You are correct! It was hidden. I deleted the C:\Users<username>\AppData\Roaming\ugsplatform and it worked! Thanks for the help!
